src/attack_flow_builder/src/assets/builder.config.validator.ts

```diff
@@ -203,11 +213,10 @@ class AttackFlowValidator extends DiagramValidator {
                     this.addError(id, "Invalid MIME Type.");
                 }
                 // Validate Payload Bin, URL, and Hashes
-                if(payloadBin?.isDefined() && url?.isDefined()) {
-                    this.addError(id, "Artifact must have either have a Payload Bin or URL, not both.");
-                }
-                if(url?.isDefined() && !hashes?.isDefined()) {
-                    this.addError(id, "Artifact URL must also have a Hash.");
+                if (payloadBin?.isDefined() && (url?.isDefined() || hashes?.isDefined()) ||
```

Comment:
The [STIX Documentation](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_4jegwl6ojbes) states that `hashes`:
> Specifies a dictionary of hashes for the contents of the **url** or the **payload_bin**.

This `if` statement doesn't allow `payload_bin` and `hashes` to be defined at the same time.